js/views/baseVw.js

```diff
@@ -150,6 +150,51 @@ export default class baseVw extends View {
     super.delegateEvents();
   }
 
+  /**
```

Comment:
As an FYI, we do have dom element caching in place in most views, but it's a more manual declarative method. For example, if I want to cache the element at `.js-orderDetailsCopiedToClipboard`, I'd implement the following method:

```
  get $copiedToClipboard() {
    return this._$copiedToClipboard ||
      (this._$copiedToClipboard = this.$('.js-orderDetailsCopiedToClipboard'));
  }
```

then in render, we'd add:

`this._$copiedToClipboard = null;`

The approach you implemented involves much less configuration and, at the very least gives us another option going forward. If we find it's working well, we could eventually refactor the other views to use it, although there'd be a little work there + we'd also need to update those views to call render in the baseVw.